src/libs/actions/Task.js

```diff
@@ -259,10 +265,39 @@ function completeTask(taskReportID, taskTitle) {
         },
     ];
 
-    // Update optimistic data for parent report action
-    const optimisticParentReportData = ReportUtils.getOptimisticDataForParentReportAction(taskReportID, completedTaskReportAction.created, CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
-    if (!_.isEmpty(optimisticParentReportData)) {
-        optimisticData.push(optimisticParentReportData);
+    // Update optimistic data and failure data for parent report action
+    if (taskReport.parentReportActionID) {
```

Comment:
Do we need this condition? I think the parent always exists

Reply:
I think so